### DISCUSSION TITLE
Production & adapter updates

### DISCUSSION THREAD

**Discussion by peterantypas** [peterantypas/maiana#92] [General]
I have started a company so I can establish a business account with the new SMT house. So I am now officially Maverick Labs LLC.

Main board SMT will start at a facility in LA sometime this month. I will still build the adapters myself until I'm comfortable handing those over as well. The solder paste and solder wire used on all boards will change from Sn42/Bi57.6/Ag0.4 to SAC305 (Sn96.5/Ag3.0/Cu0.5) which is the "gold standard" for RoHS. This also required that production main board panels are ENIG instead of HASL (more expensive, but better not to mix unknown solder into the process). 

To streamline my side of production further, I consolidated the old USB and 0183 adapters into two variants of the same basic design. Several reasons behind this:

1. I had to change 3D printing vendors, so the tolerances of the enclosure changed and the snap-in feature of the front panel was no longer reliable
2. The slide switches I used to source became unavailable and only their expensive gold contact variants can be sourced
3. The old USB adapter design required 2-side SMT plus a through-hole part, which costs more than the 0183 design (top SMT, bottom wave solder)
4. The assembly process required a cable which adds additional labor

Since I will still make those for a while and I don't intend to have a molten solder pot anywhere near my skin, I had to think of a way to optimize through hole soldering. So I made this:

![IMG_1599](https://user-images.githubusercontent.com/1565933/148666754-af917ac1-d8fb-4a99-a47f-ade1f1cffca8.jpg)

This simple 3D fixture shaves about 5 minutes of (my) labor from each board. That's hours of my time each month. It turns out I was spending more time securing the through hole parts onto the board upside down than actually soldering them with the iron. 

With the main board SMT outsourced and the USB and 0183 adapters now sharing this design, I can spend more time on something else. Like the next product, for instance ;-)

**Comment by astuder** [comment]
Congratulations! Ah, the joys of "mass" production! :)

I like that soldering jig. Making fixtures is the main reason I'm thinking about getting a 3D printer.
